tests/test_libzim.py

```diff
@@ -140,3 +140,24 @@ def test_check_mandatory_metadata(tmpdir):
             title="Test Zim",
         )
         assert zim_creator.mandatory_metadata_ok()
+
+
+def test_segfault_on_realloc(tmpdir):
+    """ assert that we are able to reallocate var from un un-close creator #31 """
+    creator = Creator(str(tmpdir / "1.zim"), "welcome", "eng", 2048)
+    # following used to segfault
+    creator = Creator(str(tmpdir / "2.zim"), "welcome", "eng", 2048)
```

Comment:
It worth to mention that we don't really "reuse" a variable here.
The second `creator = Creator(...)` create a second `Creator` and associate the name "creator" to it. So the first `Creator` is associated with no name and so the gc may delete it (without calling `close`).

This issue should be throwable with :
```
creator = Creator(...)
del creator
```
(We can even force the delete of the unassociated object with `gc.collect()`)

However, this is just for information, it is not a blocker for this PR at all.

Reply:
Agrees, following the issue's example but it's not required. Fixed it